qp_shogun/shogun/shogun.py

```diff
@@ -188,93 +187,106 @@ def shogun(qclient, job_id, parameters, out_dir):
-            success, msg = _run_commands(
-                qclient, job_id, func_cmd, sys_msg, 'Shogun functional')
-            if not success:
-                return False, None, msg
-        # Step 6 functional profile
-        sys_msg = "Step 7 of 7: Converting results to BIOM (%d/{0})"
-        # Converting redistributed files to biom
-        redist_levels = ['genus', 'species', 'strain']
-        for redist_fp, level in zip(redist_fps, redist_levels):
-            biom_in = ["redist", None, '', True]
-            output = run_shogun_to_biom(
-                redist_fp, biom_in, out_dir, level, 'redist')
-            aname = 'Taxonomic Predictions - %s' % level
-            ainfo.append(ArtifactInfo(aname, 'BIOM', [(output, 'biom')]))
-        # Coverting funcitonal files to biom
-        func_db_fp = shogun_db_functional_parser(parameters['database'])
-        for level in levels:
-            func_to_biom_fps = [
-                ["kegg.modules.coverage", func_db_fp['module'],
-                 'module', False],
-                ["kegg.modules", func_db_fp['module'], 'module', False],
-                ["kegg.pathways.coverage", func_db_fp['pathway'],
-                 'pathway', False],
-                ["kegg.pathways", func_db_fp['pathway'], 'pathway', False],
-                ["kegg", func_db_fp['enzyme'], 'enzyme', True],
-                ["normalized", func_db_fp['enzyme'], 'pathway', True]]
-
-            for biom_in in func_to_biom_fps:
-                biom_in_fp = join(func_fp, "profile.%s.%s.txt"
-                                  % (level, biom_in[0]))
-                output = run_shogun_to_biom(biom_in_fp, biom_in, out_dir,
-                                            level, 'func')
-                if biom_in[0] == 'kegg.modules.coverage':
-                    atype = 'KEGG Modules Coverage'
-                elif biom_in[0] == 'kegg.modules':
-                    atype = 'KEGG Modules'
-                elif biom_in[0] == 'kegg.pathways.coverage':
-                    atype = 'KEGG Pathways Coverage'
-                elif biom_in[0] == 'kegg.pathways':
-                    atype = 'KEGG Pathways'
-                elif biom_in[0] == 'kegg':
-                    atype = 'KEGG'
-                elif biom_in[0] == 'normalized':
-                    atype = 'Normalized'
-                else:
-                    # this should never happen but adding for completeness
-                    return False, None, "Not a valid format: %s" % biom_in[0]
-                aname = 'Functional Predictions - %s, %s' % (level, atype)
-                ainfo.append(ArtifactInfo(aname, 'BIOM', [(output, 'biom')]))
+        sys_msg = "Step 5 of 5: Converting output to BIOM"
+        qclient.update_job_step(job_id, msg)
+        output = run_shogun_to_biom(profile_fp, [None, None, None, True],
+                                    out_dir, 'shogun')
+
+        ainfo = [ArtifactInfo('Shogun Alignment', 'BIOM', [(output, 'biom')])]
```

Comment:
'Shogun Alignment' might be a misleading term. Most accurately it's a taxonomic profile, with the following ones being 'regrouped taxonomic profiles.' But 'Shogun Alignment Profile' might suffice to clearly distinguish it from the 'Shogun Taxonomic Profile' while still making it clear that it's not *just* the alignment (which we discard). What do you think?

Reply:
K, gonna use Shogun Alignment Profile